packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -721,14 +721,14 @@ void main() {
       expect(date, DateTime(2018, 1, 1, 11, 59));
 
       // 11:59 -> 9:59
-      await tester.drag(find.text('11'), -_kRowOffset * 2);
+      await tester.drag(find.text('11'), -_kRowOffset * 1.5);
```

Comment:
why do we change both the multiplier and the value? this seems to suggest the math is wrong. I don't understand why we change both.